### ISSUE TITLE
Group notifications

### ISSUE THREAD

**Issue by delusho** [TheBLVD/mammoth#107]
Could you please group notifications?

basically all likes / boosts of the same toot should be grouped together. Right now the notification tab shows more than 899 new notifications and needs to be refreshed a couple of times to get up to date.

**Comment by bdecrem** [comment]
yup, strong candidate for Mammoth 2, a few months from now. will let you know when we get to this

**Comment by delusho** [comment]
Super! That’s fantastic ! Looking very much forward to not having to refresh 5-6 times or so to reach the most current notifications.

**Comment by bdecrem** [comment]
Whoa … how popular are you??

**Comment by delusho** [comment]
Not really - it’s just 100 boosts + 100 hearts is already 200 notifications. Multiple toots receiving notifications quickly ends up in 1000 new notifications if you open the app not too regularly.

**Comment by bdecrem** [comment]
I want to go follow you! What’s your handle?

**Comment by bersfo** [comment]
Second this one! Current notifications screen scrolls forever.
